src/NzbDrone.Core/ImportLists/Simkl/User/SimklUserImport.cs

```diff
@@ -17,7 +17,7 @@ public class SimklUserImport : SimklImportBase<SimklUserSettings>
         {
         }
 
-        public override string Name => "Simkl User Watchlist";
```

Comment:
This isn't needed if this is DRYed up.

Reply:
If this line of code is removed, how is the button implemented? Currently we have two buttons: "Simkl User Watchlist" and "Simkl Anime WatchList". Not sure where to go from there.

Reply:
The renaming isn't necessary as setting the list type will be handled by a setting in `SimklUserSettings`